doc/releases/migration-guide-3.6.rst

```diff
@@ -50,6 +50,8 @@ Bluetooth
   Devicetree chosen is now ``zephyr,bt-hci-ipc``. The existing sample has also
   been renamed, from ``samples/bluetooth/hci_rpmsg`` to
   ``samples/bluetooth/hci_ipc``.
+* The BT GATT callback list is no longer cleared on :c:func:`bt_disable`. Callbacks
+  should no longer be re-registered after disabling the Bluetooth stack.
```

Comment:
I have edit suggestion for a slightly longer text that I think gives a clearer migration strategy.

```suggestion
* The :c:func:`bt_gatt_cb_register` registrations are no longer cleared on :c:func:`bt_enable`. This affects users of :c:func:`bt_disable`, who must update their code to never call :c:func:`bt_gatt_cb_register` twice with the same argument. It's now possible and recommended to instead make the call to :c:func:`bt_gatt_cb_register` before the first call to :c:func:`bt_enable`.
```

Reply:
@henrikbrixandersen